### DISCUSSION TITLE
Loading directly to your phone.

### DISCUSSION THREAD

**Discussion by ghost** [LeoKlaus/plappa#175] [Feature Requests]
While I love having a streaming server to access my audiobook library from, I like being able to load content directly onto my phone via the Finder or an app called imazing.

**Comment by LeoKlaus** [comment]
Hey Maplebiscuits,

this is not really the intended use-case for plappa, as it doesn't store or load any metadata on its own.
The intended way of doing this would be to add the book to your server and the download it to your phone through plappa.

If anything is unclear, feel free to comment :)
